### ISSUE TITLE
"No such file or directory: '/sys/class/backlight/acpi_video0/max_brightness" kills the whole i3pystatus output

### ISSUE THREAD

**Issue by teto** [enkore/i3pystatus#442]
```
Traceback (most recent call last):
  File "/home/teto/.local/bin/i3pystatus", line 9, in <module>
    load_entry_point('i3pystatus', 'console_scripts', 'i3pystatus')()
  File "/home/teto/i3pystatus/i3pystatus/__init__.py", line 47, in main
    imp.load_source(module_name, args.config)
  File "/usr/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/teto/.i3/myStatus.py", line 51, in <module>
    status.register("backlight",   format="{percentage}%",)
  File "/home/teto/i3pystatus/i3pystatus/core/__init__.py", line 109, in register
    return self.modules.append(module, *args, **kwargs)
  File "/home/teto/i3pystatus/i3pystatus/core/util.py", line 73, in append
    module, *args, **kwargs)
  File "/home/teto/i3pystatus/i3pystatus/core/imputil.py", line 47, in instanciate_class_from_module
    return self.instanciate_class_from_module(self.get_module(module), *args, **kwargs)
  File "/home/teto/i3pystatus/i3pystatus/core/imputil.py", line 45, in instanciate_class_from_module
    return self.get_class(module)(*args, **kwargs)
  File "/home/teto/i3pystatus/i3pystatus/core/modules.py", line 64, in __init__
    super(Module, self).__init__(*args, **kwargs)
  File "/home/teto/i3pystatus/i3pystatus/core/settings.py", line 109, in __init__
    self.init()
  File "/home/teto/i3pystatus/i3pystatus/backlight.py", line 50, in init
    with open(self.base_path + maxbfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/sys/class/backlight/acpi_video0/max_brightness'
```

At first I had not installed xbacklight and I had a "module error", then I installed xbacklight and got an the uncaught past error (from ".xsession-errors") which kills the whole i3pystatus "status_command process exited unexpectedly". It would be nice to catch it so that it doesn't kill the whole status.

**Comment by enkore** [comment]
Yeah, this should be handled better.

Root cause is probably a different name in the /sys tree. This once changed for me with a kernel update even though the hardware was the same (ugh)

**Comment by teto** [comment]
I pulled the changes but it still kills the whole output as
"/sys/class/backlight" is empty. The error should remain within the module
to help the user find out which module fails

2016-10-23 23:28 GMT+02:00 enkore <notifications@github.com>:

**Comment by hcpl** [comment]
@teto, does this fix for you: #483?

**Comment by teto** [comment]
Looks good I have a "no backlight" message display instead of a ruined display thanks :)
You can close definitely when it gets merged
